shared/src/userData.ts

```diff
@@ -48,6 +49,7 @@ export interface Preferences {
   roadmapOpenSteps: number[];
   hiddenFundingIds: string[];
   hiddenCertificationIds: string[];
+  showSuccessfulRegistrationCard: boolean | undefined;
```

Comment:
Instead of a boolean, I think this would be better as a list of ids for hidden sidebar cards.